demo/src/plugins/dashboard/components/top-articles/index.js

```diff
@@ -33,18 +34,24 @@ export default function TopArticles(props) {
             <tr>
               <th>Title</th>
               <th className="px-3"></th>
+              <th></th>
               <th>Slotting</th>
               <th>Order Date</th>
               <th></th>
             </tr>
             </thead>
             <tbody>
             {response.get('nodes', []).map(node => {
+              const isLocked = node.get('is_locked');
+              if (isLocked && !canUnlock) {
```

Comment:
@mtakla16 remove this check, users who can't unlock will never see even get these records (we need to make a LockableEnricher to enrich the search articles request).  ref 0.x version and adapt to new listener/enricher strategy.  lmk if u have questions